docs/docs.html

```diff
@@ -632,91 +643,185 @@ <h4>Example of icon buttons</h4>
             
                   &lti class="material-icons icon-only">notifications&lt/i>
                 </code></pre>
-
           </div>
         </div>
       </div>
 
-      <!-- card section starts -->
-      <div class="card-section" id="card">
+      <hr />
+      <!-- --------Input section starts----- -->
+
+      <div class="input-section" id="input">
         <div class="heading p-xs mb-sm">
-          <h2>Cards</h2>
-          <h5>cards are important part od many ecommerce websites</h5>
+          <h2>Input</h2>
+          <h5>Various Types of input used</h5>
         </div>
 
         <div class="description-box mb-sm">
           <h3>Example</h3>
           <div class="text-description p-xs mb-sm">
-            <p>Simple card/p></p>
+            <p>Simple input</p>
           </div>
-        </div>
 
-        <div class="component-container">
-          <div class="card-container one">
-            <img
-              class="card-image"
-              src="/assets/tshirt-image.jpg"
-              alt="T-shirt image"
-            />
+          <div class="component-container component-container-input">
+            <div class="input-holder">
+              <label class="label" for="username">Username </label>
+              <input
+                id="username"
+                placeholder="type username here"
+                type="text"
+                required
+              />
+              <span></span>
+            </div>
+
+            <div class="input-holder">
+              <label class="label" for="email">Email </label>
+              <input type="email" placeholder="type email here here" />
+            </div>
 
-            <div class="card-details">
-              <h2 class="card-title">AUTHENTIC MEN'S WALKOUT JERSEY</h2>
-              <h5 class="card-price">$80.00</h5>
+            <div class="input-holder">
+              <label for="pwd">Password:</label>
+              <input type="password" id="pwd" name="pwd" />
             </div>
           </div>
 
-          <div class="card-container one">
-            <img
-              class="card-image"
-              src="/assets/tshirt-image.jpg"
-              alt="T-shirt image"
-            />
+          <div class="code-snippets">
+            <pre><code class="html">&lt;div class=&quot;input-holder&quot;&gt;
```

Comment:
Indent it properly. To remove white spaces from the code block you can style the code tag either by giving white-space:pre-line/pre-wrap or by giving a suitable negative left margin.

Reply:
Will add this new patch